apollo-ios/Sources/Apollo/ApolloStore.swift

```diff
@@ -15,13 +13,66 @@ public protocol ApolloStoreSubscriber: AnyObject {
   ///   - store: The store which made the changes
   ///   - changedKeys: The list of changed keys
   ///   - contextIdentifier: [optional] A unique identifier for the request that kicked off this change, to assist in de-duping cache hits for watchers.
+  /// @deprecated
```

Comment:
Is this a Swift Macro you've defined somewhere else? We'll need to change this to use the `@available` attribute instead.